packages/ui/src/theme/tokens/components/aiConversation.ts

```diff
@@ -0,0 +1,120 @@
+    assistant?: DesignTokenProperties<'backgroundColor', OutputType>;
+  };
+  form?: DesignTokenProperties<'gap' | 'padding', OutputType>;
+  attachment?: DesignTokenProperties<
+    | 'borderColor'
+    | 'borderWidth'
+    | 'borderRadius'
+    | 'fontSize'
+    | 'paddingBlock'
+    | 'paddingInline'
+    | 'gap',
+    OutputType
+  > & {
+    list?: DesignTokenProperties<'paddingBlockStart' | 'gap', OutputType>;
+    image?: DesignTokenProperties<'width' | 'height', OutputType>;
+    size?: DesignTokenProperties<
+      'color' | 'fontSize' | 'fontWeight',
+      OutputType
+    >;
+    name?: DesignTokenProperties<
+      'color' | 'fontSize' | 'fontWeight',
+      OutputType
+    >;
+    remove?: DesignTokenProperties<'padding', OutputType>;
+  };
+}
+
+export const aiConversation: Required<AIConversationTokens<'default'>> = {
+  message: {
+    backgroundColor: { value: '{colors.background.secondary.value}' },
+    borderRadius: { value: '{radii.large.value}' },
+    gap: { value: '{space.small.value}' },
+    paddingBlock: { value: '{space.small.value}' },
+    paddingInline: { value: '{space.small.value}' },
+    user: {
+      backgroundColor: { value: '{colors.background.secondary.value}' },
+    },
+    assistant: {
+      backgroundColor: { value: '{colors.primary.10.value}' },
+    },
+    sender: {
+      gap: { value: '{space.small.value}' },
+      username: {
+        color: { value: '{colors.font.primary.value}' },
+        fontSize: { value: 'inherit' },
+        fontWeight: { value: '{fontWeights.bold.value}' },
+      },
+      timestamp: {
+        color: { value: '{colors.font.tertiary.value}' },
+        fontSize: { value: 'inherit' },
+        fontWeight: { value: 'inherit' },
+      },
+    },
+    body: { gap: { value: '{space.xs.value}' } },
+  },
+
+  form: {
+    gap: { value: '{space.small.value}' },
+    padding: { value: '{space.small.value}' },
+  },
+
+  attachment: {
+    borderColor: { value: '{colors.border.secondary.value}' },
+    borderWidth: { value: '{borderWidths.small.value}' },
+    borderRadius: { value: '{radii.small.value}' },
+    fontSize: { value: '{fontSizes.small.value}' },
+    paddingBlock: { value: '{space.xxxs.value}' },
+    paddingInline: { value: '{space.xs.value}' },
+    gap: { value: '{space.xs.value}' },
+
+    list: {
+      paddingBlockStart: { value: '{space.xs.value}' },
+      gap: { value: '{space.xxs.value}' },
+    },
+
+    name: {
+      color: { value: '{colors.font.primary.value}' },
+      fontSize: { value: '{fontSizes.small.value}' },
+      fontWeight: { value: '{fontWeights.normal.value}' },
+    },
+
+    size: {
+      color: { value: '{colors.font.tertiary.value}' },
+      fontSize: { value: '{fontSizes.small.value}' },
+      fontWeight: { value: '{fontWeights.normal.value}' },
+    },
+
+    remove: {
+      padding: { value: '{space.xxs.value}' },
+    },
+
+    image: {
+      width: { value: '{fontSizes.medium.value}' },
```

Comment:
Why would the image size match the font sizes?

Reply:
The image here is the image inside the attachment so we want it to be the size of the font size:

![CleanShot 2024-10-10 at 13 49 27@2x](https://github.com/user-attachments/assets/1126cf97-0fb3-4790-85f4-d34eb1c549eb)